client/src/app/components/edit-profile/edit-profile.component.scss

```diff
@@ -0,0 +1,85 @@
+@import "../../../styles.scss";
+.wrapper {
+    max-width: 575px;
+    margin-left: auto;
+    margin-right: auto;
+    .cover-picture {
+        width: calc(100%-10px);
+        height: 350px;
+        background: center;
+        background-image: url(https://i.pinimg.com/originals/d7/0b/19/d70b19d883f6d08caccef02b8479fbbb.png);
+        position: relative;
+
+        .dot {
+            height: 27px;
+            width: 27px;
```

Comment:
buttons need to be at least 40px to click on them comfortably, 27px is too small